server/controllers/availability.js

```diff
@@ -0,0 +1,115 @@
+const { Availability, Schedule } = require("../models/Availability");
+const asyncHandler = require("express-async-handler");
+
+// @route POST /availability/createSchedule
```

Comment:
Let's update this comment to be `@route POST /availability/schedule`

My other comment will go into changing up the routing to make things a bit more clear on what the resource does.